standard/namespace.lua

```diff
@@ -32,14 +32,14 @@ function Namespace.idFromName(name)
 end
 
 ---Fetches the namespace name for a given namespace id (number)
----@param id integer?
+---@param id integer|string|nil
```

Comment:
we are doing a `tonumber` so we technically accept numeric (integer) strings
for other strings it will just return nil just like for non valid numbers